share/functions/realpath.fish

```diff
@@ -6,8 +6,10 @@
 # If we see that there is an external command by that name we just return. That will cause fish to
 # run the external command. On the other hand, if an external command isn't found we define a
 # function that will provide fallback behavior.
-if not type -q -P realpath
-    function realpath --description 'fallback realpath implementation'
-        builtin fish_realpath $argv[-1]
-    end
+if not command -v realpath > /dev/null
+	if command -v grealpath > /dev/null
+		alias realpath=grealpath
```

Comment:
As @faho pointed out I also agree this should explicitly define the function rather than using `alias`. Look at the description for the `alias` function.